src/components/structures/RoomView.js

```diff
@@ -1595,6 +1595,7 @@ module.exports = React.createClass({
                 atEndOfLiveTimeline={this.state.atEndOfLiveTimeline}
                 sentMessageAndIsAlone={this.state.isAlone}
                 hasActiveCall={inCall}
+                isPeeking={myMembership !== "join"}
```

Comment:
Would it be an option to just not render the whole thing if you're not joined?

Reply:
The status bar also handles things like the jump to bottom button, so it's not as easy as just taking it out.

Reply:
Gotcha :+1: